modules/api.js

```diff
@@ -0,0 +1,51 @@
+const loadPage = async() => {
+  try {
+    const response = await fetch(
+      'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/c2i7GgLdDFFaD5JEzNMX/scores/',
+      {
+        method: 'POST',
+        body: JSON.stringify(),
+        header: {
+          'Content-type': 'application/json; charaset=UTF-8',
+        },
+      },
+    );
+    if(response.ok) {
+      console.log(response);
+      const section = {
+        img: '../src/images/pic1.jpg',
```

Comment:
In order to follow the project's requirements, pull the cards info (pics, title) from the API.